### ISSUE TITLE
cutoff-value range

### ISSUE THREAD

**Issue by ineshotopp** [shapiromatron/bmds-archive#65]
The allowable range for the BDM cutoff-value on benchmarkdose.com only works for increasing dose-response data because of the code
```python
 if not domain[0] < self.adversity_value < domain[1]:
                raise ValueError(
                       'Adversity value not in allowable domain: {} ({} - {})'
                       .format(self.adversity_value, domain[0], domain[1]))
```
in base.py. Instead
```python
 if not min(domain) < self.adversity_value < max(domain):
                raise ValueError(
                       'Adversity value not in allowable domain: {} ({} - {})'
                       .format(self.adversity_value, min(domain), max(domain))
```
might work?
Thanks.

**Comment by shapiromatron** [comment]
@ineshotopp thanks for digging into this code!  

This source code isn't used on benchmarkdose.com; that's a separate code base that's owned and operated by  Kan Shao, https://benchmarkdose.com/about/.

This python package uses the US EPA benchmark dose modeling tools, https://www.epa.gov/bmds. We think it supports decreasing values, can you confirm that the issue you saw was with this code, and not the other website?

**Comment by ineshotopp** [comment]
Thanks for the quick reaction. I used the web-based system on https://benchmarkdose.com and thats were I ran into the problem. I tried to dig into it and apparently found the wrong source code. Sorry for that.  
I downloaded the Excel-tool from https://www.epa.gov/bmds. Using the BMR Type "Point" works fine. I get vastly different results compared to using PROAST, but that is either me not getting it right yet or the seemingly slightly differing model equations.
So with the analysis using https://benchmarkdose.com, I will try to get a response by Kan Shao or someone else working with it. Thanks.
